src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -7,13 +7,27 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <!-- Although we have a NS configuration, we know we use API that's not currently whitelisted: just validate against OneCore -->
     <UWPCompatible>false</UWPCompatible>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netfx'">true</IsPartialFacadeAssembly>
+    <GeneratePlatformNotSupportedAssembly Condition="'$(TargetsWindows)' != 'true' AND '$(TargetsUnix)' != 'true'">true</GeneratePlatformNotSupportedAssembly>
```

Comment:
I assume this condition is about getting all the configurations where you didn't specify an OS?